docs/Team/Working-Remote.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: "Tallwave Guidelines — Team — Working Remote"
+---
+
+[&larr; Home]({{ site.root }}/) &mdash; [Team]({{ site.root }}/Team)
+
+# Our remote environment
+
+Tallwave as an organization is primarily on-site, but we have a handful of remote employees, as well as a strong culture of working from home when you need to go heads-down on a problem. Currently, the dev team is ~20% full-time remote. Given that, async communication is very important, for which we rely on Basecamp. Email is used mainly for company announcements and project-client communication, and video calls are for anything from client meetings to 1-on-1s to pair programming.
```

Comment:
Specify that the 20% number is FTEs. Also mention that the majority of our contractors are remote and so it is important to have solid remote protocols.